Cargo.toml

```diff
@@ -13,6 +13,8 @@ rustdoc-types = "0.10.0"
 serde = { version = "1.0.135", features = ["derive"] }
 serde_json = "1.0.77"
 thiserror = "1.0.29"
+syntect = "4.6.0"
+lazy_static = "^1"
```

Comment:
Would you say it is too early for me to look into this draft PR in some detail? One comment I have after a quick skim-through is that these deps are not needed any longer, right? Sorry if this is a silly comment to make on a draft :)

Reply:
Yes those are not needed. I will make sure to run clippy all over the PR code to catch things like this.

Reply:
👍 

In case you didn't find it yet, there is a convenient script you can run locally that simulates CI: `./scripts/run-ci-locally.sh`